docs/configuration.adoc

```diff
@@ -138,6 +138,36 @@ IMPORTANT: This strategy is currently only applicable for _white box_ testing ap
 
 WARNING: At this moment, this strategy does not work with Java 9.
 
+===== Explicitly Set
+
+By default affected strategy uses _imports_ of tests to build the graph of dependencies.
```

Comment:
Maybe `graph of collaborators` - dependency is rather broad term and could be understood that we build a graph of libraries we use based on imports